client/src/www/views/servers_view/index.ts

```diff
@@ -87,7 +87,7 @@ export class ServerList extends LitElement {
       }
       footer {
         border-top: 1px solid rgba(0, 0, 0, 0.08);
-        padding: 24px 0 16px;
+        padding: 24px 0 max(16px, env(safe-area-inset-bottom));
```

Comment:
`max()` was added in Chrome 79. Not sure if it covers every supported OS version, but since the related `min()` function is already used in the project, I figured it’s fine to use `max()` here.